src/Helpers.php

```diff
@@ -156,4 +162,118 @@ public static function cleanSchemaName(string $name): string {
 		global $readableAppID;
 		return substr($name, strlen($readableAppID));
 	}
+
+	protected static function getScopeNameFromAttributeArgument(Arg $arg, string $routeName): ?string {
+		if ($arg->name->name === 'scope') {
+			if ($arg->value instanceof ClassConstFetch) {
+				if ($arg->value->class->getLast() === 'OpenAPI') {
+					return self::getScopeNameFromConst($arg->value);
+				}
+			} elseif ($arg->value instanceof String_) {
+				return $arg->value->value;
+			} else {
+				Logger::panic($routeName, 'Can not interpret value of scope provided in OpenAPI(scope: …) attribute. Please use string or OpenAPI::SCOPE_* constants');
+			}
+		}
+
+		return null;
+	}
+
+	protected static function getScopeNameFromConst(ClassConstFetch $scope): string {
+		return match ($scope->name->name) {
+			'SCOPE_DEFAULT' => 'default',
+			'SCOPE_ADMINISTRATION' => 'administration',
+			'SCOPE_FEDERATION' => 'federation',
+			'SCOPE_IGNORE' => 'ignore',
+			// Fall back for future scopes assuming we follow the pattern (cut of 'SCOPE_' and lower case)
+			default => strtolower(substr($scope->name->name, 6)),
```

Comment:
I'd rather panic in this case. If there is a new scope then it will also be added to this tool and thus the developers need to update to a newer version.

Reply:
Well that is exactly the reason why I would not do it. Maybe updating the tooling is not easily possible or comes with breaking changes in the future but a release needs to be made quickly due to time pressure/major release support/etc.

Reply:
Alright